diff -aurN libtxc_dxtn.orig/Makefile libtxc_dxtn/Makefile
--- libtxc_dxtn.orig/Makefile	2006-01-17 23:14:25.000000000 +0100
+++ libtxc_dxtn/Makefile	2006-06-16 16:20:56.275959984 +0200
@@ -1,6 +1,6 @@
 CFLAGS += -Wall -pedantic -fPIC
 OPT_CFLAGS = -O3
-LDFLAGS += -shared -fPIC
+LDFLAGS += -shared -fPIC -lpthread
 OBJS = txc_compress_dxtn.o txc_fetch_dxtn.o
 LIB = libtxc_dxtn.so
 
@@ -14,5 +14,5 @@
 	rm -f $(OBJS) $(LIB)
 
 install: $(LIB)
-	install -d $(DESTDIR)/usr/lib
-	install -m 755 $(LIB) $(DESTDIR)/usr/lib
+	install -d $(DESTDIR)$(LIBDIR)
+	install -m 755 $(LIB) $(DESTDIR)$(LIBDIR)
